src/hooks/useMediaModal.tsx

```diff
@@ -62,7 +62,7 @@ export const useMediaModal = () => {
       dismissCallback,
       retryLoading = false,
       retryLimit=5,
-      minHeight='300px',
+      minHeight='240px',
```

Comment:
While this seems OK, I wonder why we are just changing the default, seems like it only fixes this one case.  Also I wonder if actually just fixing the Thumbnail Modal is a more appropriate fix. See here:
https://github.com/future-standard/scorer-ui-kit/blob/29bcc63886fe6f08ca8850a0164cd97bc4e85ce9/src/Tables/atoms/TableRowThumbnail.tsx#L107

Reply:
@JoshLipps I had handled the issue from props as you suggested. And I reverted the change in `useMediaModal`. Please review it again.